src/containers/PharmacyList/PharmacyList.tsx

```diff
@@ -190,9 +191,14 @@ export function PharmacyList(props: Props) {
 
   return (
     <>
+      <section aria-label="search" style={{ marginBottom: "2rem" }}>
```

Comment:
Very minor, but there is an inconsistency in that this `section` element isn't in the `Home` component. If it should always be included, it should probably be in the `SearchCard` component.